### PR TITLE
GAPI (fluid): Sobel 3x3 optimization: CV_SIMD dynamic dispatching

### PR DESCRIPTION
**Pull request by elatkin**
```
force_builders=Linux AVX2
```

### CODE REVIEW

**Comment by elatkin** [comment]
This PR is addendum for https://github.com/opencv/opencv/pull/13070

File: modules/gapi/src/backends/fluid/gfluidimgproc_func.dispatch.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018 Intel Corporation
+
+#if !defined(GAPI_STANDALONE)
+
+#include "gfluidimgproc_func.hpp"
+#include "gfluidimgproc_func.simd.hpp"
+#include "gfluidimgproc_func.simd_declarations.hpp"
```
[the remark this answers is not in the document]
Comment (elatkin) [reply modules/gapi/src/backends/fluid/gfluidimgproc_func.dispatch.cpp]: Yes, this trick seems to fix the CV_SIMD dynamic dispatching. Please see the next commit.

**Comment by elatkin** [comment]
@alalek  
Could you please merge this PR?

File: modules/gapi/CMakeLists.txt [hunk 2]
```diff
@@ -69,7 +69,7 @@ set(gapi_srcs
     src/backends/fluid/gfluidbuffer.cpp
     src/backends/fluid/gfluidbackend.cpp
     src/backends/fluid/gfluidimgproc.cpp
-    src/backends/fluid/gfluidimgproc_func.cpp
```
Comment (alalek) [inline modules/gapi/CMakeLists.txt]: Looks like this file is not removed from sources: src/backends/fluid/gfluidimgproc_func.cpp

**Comment by alalek** [comment]
Perhaps need to fix G-API "standalone" build mode after this patch.

File: modules/gapi/CMakeLists.txt (hunk 2 above)
Comment (elatkin) [reply modules/gapi/CMakeLists.txt]: Yes, I forgot removing gfluidimgproc_func.cpp, thanks for finding this problem!

Comment (elatkin) [reply modules/gapi/CMakeLists.txt]: Removing this needless file with my next PR #13157

**Comment by elatkin** [comment]
> Perhaps need to fix G-API "standalone" build mode after this patch.  

I have created the new issue to track this: 
#13158 G-API: establish regular testing of "standalone" build

**Pull request merged**